src/main/java/com/LetMeDoWith/LetMeDoWith/infrastructure/task/repository/DowithTaskRepositoryImpl.java

```diff
@@ -0,0 +1,52 @@
+package com.LetMeDoWith.LetMeDoWith.infrastructure.task.repository;
+
+import com.LetMeDoWith.LetMeDoWith.domain.task.repository.DowithTaskRepository;
+import com.LetMeDoWith.LetMeDoWith.domain.task.model.DowithTask;
+import com.LetMeDoWith.LetMeDoWith.infrastructure.task.jpaRepository.DowithTaskConfirmJpaRepository;
+import com.LetMeDoWith.LetMeDoWith.infrastructure.task.jpaRepository.DowithTaskJpaRepository;
+import com.LetMeDoWith.LetMeDoWith.infrastructure.task.jpaRepository.DowithTaskRoutineJpaRepository;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@RequiredArgsConstructor
+public class DowithTaskRepositoryImpl implements DowithTaskRepository {
+
+  private final DowithTaskJpaRepository dowithTaskJpaRepository;
+  private final DowithTaskRoutineJpaRepository dowithTaskRoutineJpaRepository;
+  private final DowithTaskConfirmJpaRepository dowithTaskConfirmJpaRepository;
+
+  @Override
+  public Optional<DowithTask> getDowithTask(Long id, Long memberId) {
+    return dowithTaskJpaRepository.findDowithTaskAggregate(id, memberId);
+  }
+
+  @Override
+  public List<DowithTask> getDowithTasks(Long memberId, LocalDate date) {
+    return dowithTaskJpaRepository.findDowithTaskAggregates(memberId, date);
+  }
+
+  @Override
+  public List<DowithTask> getDowithTasks(Long memberId, Set<LocalDate> dates) {
+    return dowithTaskJpaRepository.findDowithTaskAggregates(memberId, dates);
+  }
```

Comment:
https://github.com/LetMeDoWith/LetMeDoWith-BE/pull/23#discussion_r1807353510

이 리뷰에 의해서
```suggestion
  @Override
  public List<DowithTask> getDowithTasks(Long memberId, LocalDate date) {
    return dowithTaskJpaRepository.findAllDowithTaskAggregates(memberId, date);
  }

  @Override
  public List<DowithTask> getDowithTasks(Long memberId, Set<LocalDate> dates) {
    return dowithTaskJpaRepository.findAllDowithTaskAggregates(memberId, dates);
  }
```
로 수정해야 할 것 같습니다.

Reply:
넵넵 감사합니다 기능동작에는 이상없지만 컨벤션에 맞게 수정했습니다